test/suite/unit/Output/Copy/Serializer/QuerySerializerTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoclerLabs\ApiClientGenerator\Test\Unit\Output\Copy\Serializer;
+
+use DateTimeImmutable;
+use DateTimeZone;
+use DoclerLabs\ApiClientGenerator\Output\Copy\Request\RequestInterface;
+use DoclerLabs\ApiClientGenerator\Output\Copy\Schema\SerializableInterface;
+use DoclerLabs\ApiClientGenerator\Output\Copy\Serializer\QuerySerializer;
+use JsonSerializable;
+use PHPUnit\Framework\TestCase;
+
+class QuerySerializerTest extends TestCase
+{
+    private QuerySerializer $sut;
+
+    public function setUp(): void
+    {
+        $this->sut = new QuerySerializer();
+    }
+
+    public function testSerialization(): void
+    {
+        $serializable = $this->createMock(SerializableInterface::class);
+        $serializable->expects($this->once())
+            ->method('toArray')
+            ->willReturn(['foo' => 'bar']);
+
+        $jsonSerializable = $this->createMock(JsonSerializable::class);
+        $jsonSerializable->expects($this->once())
```

Comment:
current UTs are using self:: for the static methods from phpunit like `once` and `assert*`.

Reply:
Changed to static methods.